crates/holoom_recipe/coordinator/tests/config.rs

```diff
@@ -129,9 +129,9 @@ impl TestSetup {
         self.cells[1].agent_pubkey().clone()
     }
 
-    pub fn bob_pubkey(&self) -> AgentPubKey {
-        self.cells[1].agent_pubkey().clone()
-    }
+    //pub fn bob_pubkey(&self) -> AgentPubKey {
+    //    self.cells[1].agent_pubkey().clone()
+    //}
 
     pub async fn consistency(&self) {
         await_consistency(Duration::from_secs(60), self.cells.iter())
```

Comment:
Why is the test setup coded repeated?

Reply:
i did try to put the common test config code into shared utils / shared dir .. but i could not get it to work with dev-dependencies outside of a test context unfortunately...      with module / microservices one has to compromise on DRY .. often like 20%